data/yam/agama/auto/template.jsonnet

```diff
@@ -0,0 +1,12 @@
+local lib = import 'lib/base.libsonnet';
+
+function(storage='default', product='SLES', scripts='default', encrypted=false) {
```

Comment:
we have right now a case for "import profile" that add only scripts, so in a way `user` and `root` are also optional.